### PR TITLE
test: casting numbers inside factory functions

### PR DESCRIPTION
**Pull request by fritzschoff**
Screenshot 2024-06-11 at 15 35 44

### CODE REVIEW

File: protocol/synthetix/subgraph/tests/event-factories.ts [hunk 1]
```diff
@@ -600,29 +663,115 @@ export function createVaultLiquidationEvent(
...
   newVaultLiquidationEvent.parameters.push(
-    new ethereum.EventParam('sender', ethereum.Value.fromAddress(sender))
+    new ethereum.EventParam('sender', ethereum.Value.fromAddress(Address.fromString(sender)))
   );
   newVaultLiquidationEvent.block.timestamp = BigInt.fromI64(block['timestamp']);
   newVaultLiquidationEvent.block.number = BigInt.fromI64(block['blockNumber']);
   return newVaultLiquidationEvent;
 }
+
+export function createMarketRegisteredEvent(
```
Comment (noisekit) [inline protocol/synthetix/subgraph/tests/event-factories.ts:696]: Instead pf moving all into one file its the opposite: split from one big file into separate per generator

File: protocol/synthetix/subgraph/tests/handleCollateralConfigured.ts [hunk 2]
```diff
@@ -8,25 +8,25 @@ export default function test(): void {
...
   const newCollateralConfiguredEvent = createCollateralConfiguredEvent(
-    Address.fromString(address),
+    address,
     true,
-    BigInt.fromI32(200),
-    BigInt.fromI32(50),
-    BigInt.fromI32(90),
-    Bytes.fromByteArray(Bytes.fromBigInt(BigInt.fromI32(10))),
-    BigInt.fromI32(500),
+    200,
```
Comment (noisekit) [inline protocol/synthetix/subgraph/tests/handleCollateralConfigured.ts:13]: Can we have variable names please as unnamed function arguments when they are longer than one are confusing

File: protocol/synthetix/subgraph/tests/handlePoolConfigurationSet.ts [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import { assert } from 'matchstick-as';
 import { ethereum, store } from '@graphprotocol/graph-ts';
 import { handleMarketCreated, handlePoolConfigurationSet, handlePoolCreated } from '../mainnet';
-import { createPoolConfigurationSetEvent, createPoolCreatedEvent } from './event-factories';
-import { createMarketRegisteredEvent } from './event-factories/createMarketRegisteredEvent';
+import {
+  createPoolConfigurationSetEvent,
+  createPoolCreatedEvent,
+  createMarketRegisteredEvent,
+} from './event-factories';
```
Comment (noisekit) [inline protocol/synthetix/subgraph/tests/handlePoolConfigurationSet.ts:8]: Quite tho opposite of the original intention to have each in a separate file that I haven't had enough time to refactor last time (tho all the new code in markets follows)

**Review by noisekit — COMMENTED** [review]
Restoring my faith in humanity brother

appreciating the come back a lot!

**Comment by fritzschoff** [comment]
each factory has its own file now

**Review by noisekit — APPROVED** [review]
amazing

**Pull request merged**